src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine fv122 = new Excavator();
+        Machine opc47 = new Bulldozer();
+        Machine skania = new Truck();
+        Machine[] buildingMachines = new Machine[] {fv122, opc47, skania};
+        for (int i = 0; i < buildingMachines.length; i++) {
+            buildingMachines[i].doWork();
+        }
+        System.out.println("Machines working all day and then...\n");
```

Comment:
The message printed to the console should be more informative about the type of machines that have been working. Consider specifying which machines were working, such as 'Excavator, Bulldozer, and Truck have been working all day and then...'. This would adhere to the checklist requirement for informative messages in methods.